### ISSUE TITLE
Issue with validation set for cifarfs?

### ISSUE THREAD

**Issue by brando90** [WangYueFt/rfs#33]
my SL curves look flat, but there isn't anything fancy I am doing. I am just using the dataloaders you provide. Did you experience this?
![train_eval](https://user-images.githubusercontent.com/1855278/153481059-412bf4d3-4ffb-4d04-8e95-103b258ca0ab.png)

@WangYueFt @HobbitLong

**Comment by brando90** [comment]
this doesn't seem to be the case with mini-imagenet. Perhaps the data for cifar-fs you provide has a bug? @HobbitLong @WangYueFt 
```
====> about to start train loop
Starting training...
  3% (7 of 200) |          | Elapsed Time: 0:04:06 | ETA:   1:53:27 |  35.5 s/it
epoch_num=0: train_loss=tensor(4.4372, device='cuda:0', grad_fn=<NllLossBackward>), train_acc=tensor(0.0137, device='cuda:0')
epoch_num=0: val_loss=tensor(4.1640, device='cuda:0'), val_acc=tensor(0.0078, device='cuda:0')
epoch_num=1: train_loss=3.631721493403117, train_acc=0.11572916805744171
epoch_num=1: val_loss=tensor(3.9233, device='cuda:0'), val_acc=tensor(0.1016, device='cuda:0')
epoch_num=2: train_loss=3.089755102793376, train_acc=0.21927084028720856
epoch_num=2: val_loss=tensor(3.3963, device='cuda:0'), val_acc=tensor(0.1406, device='cuda:0')
epoch_num=3: train_loss=2.7695435428619386, train_acc=0.2871614694595337
epoch_num=3: val_loss=tensor(3.2302, device='cuda:0'), val_acc=tensor(0.2031, device='cuda:0')
epoch_num=4: train_loss=2.4909704526265464, train_acc=0.3484114706516266
epoch_num=4: val_loss=tensor(3.4514, device='cuda:0'), val_acc=tensor(0.1719, device='cuda:0')
epoch_num=5: train_loss=2.226396697362264, train_acc=0.4116145968437195
epoch_num=5: val_loss=tensor(2.9306, device='cuda:0'), val_acc=tensor(0.2734, device='cuda:0')
epoch_num=6: train_loss=1.9944572687149047, train_acc=0.4642968773841858
epoch_num=6: val_loss=tensor(2.9128, device='cuda:0'), val_acc=tensor(0.3281, device='cuda:0')
epoch_num=7: train_loss=1.7948959430058797, train_acc=0.5123437643051147
epoch_num=7: val_loss=tensor(2.6068, device='cuda:0'), val_acc=tensor(0.3516, device='cuda:0'
```

**Comment by brando90** [comment]
ok the validation and test are in a different range of indices so of course it will always predict WORSE than random. 

i.e. the output of the net is trained on 64 so its very unlikely to predict something in the range given for the validation and test so it predicts worse than chance.

**Comment by brando90** [comment]
https://datascience.stackexchange.com/questions/22194/is-it-possible-for-a-ml-model-to-perform-worse-than-random-classification/108061#108061
